src/main/resources/i18n/messages.properties

```diff
@@ -11,7 +11,7 @@ error.message=Message:
 email.activation.title=Artemis account activation
 email.activation.greeting=Dear {0}
 email.activation.text1=Your Artemis account has been created, please click on the link to activate it:
-email.activation.text2=Regards,
+email.activation.text2=Regards
```

Comment:
A quick google search told me that a comma is actually correct here.

Reply:
![image](https://user-images.githubusercontent.com/65814168/142205509-934c9d64-60dd-411f-9a54-108ab16bba9b.png)
I guess I got confused. I assume you have no comma in the German version but you need one in the English version. Ok I added it back for the English one